src/commands/diff.ts

```diff
@@ -0,0 +1,109 @@
+import { flags } from '@oclif/command';
+import * as diff from '@asyncapi/diff';
+import * as parser from '@asyncapi/parser';
+import { load } from '../models/SpecificationFile';
+import Command from '../base';
+import { ValidationError } from '../errors/validation-error';
+import { SpecificationFileNotFound } from '../errors/specification-file';
+
+export default class Diff extends Command {
+  static description = 'find diff between two asyncapi files';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    format: flags.string({
+      char: 'o',
+      description: 'output format',
```

Comment:
what other formats are supported? tried yaml and failed

Reply:
also there is no help message when I use other formats, like `not supported, this is the list we support atm: bla bla`

Reply:
got it. 
Also, this made me wonder, right now only JSON output is supported, but we have plan for adding HTML/YAML/etc. types of output. 
Means we need to convert the JSON data to those formats. I was wondering whether we should have this "convert" function in diff library itself or we implement those standalone in the CLI?

Reply:
imho this is for diff library to have this functionality of different outputs

Reply:
Got it. I will start working on different output formats after this PR.